test/manifest.ts

```diff
@@ -829,6 +850,674 @@ describe('Manifest', () => {
     });
   });
 
+  describe('githubRelease', () => {
+    it('python draft and node package', async () => {
```

Comment:
nit: mind making these read a bit better, e.g.,

`it('handles node with draft false python with draft true`)`

I realize I'm being a little pedantic.